jujubundlelib/validation.py

```diff
@@ -2,11 +2,12 @@
 # Licensed under the AGPLv3, see LICENCE file for details.
 
 from __future__ import unicode_literals
+from __future__ import absolute_import
```

Comment:
Similar minor suggestions for import syntax here.

Reply:
yup, will fix.